index.html

```diff
@@ -41,6 +42,7 @@
 <script src="libs/mode-solidity.js"></script>
 <script src="bin/soljson-latest.js"></script>
 <script src="libs/ethereumjs-vm.js"></script>
+<script src="libs/universal-dapp.js"></script>
 <script src="libs/web3.min.js"></script>
```

Comment:
I think the only thing I changed was exporting `web3.eth.function`, but it might be that this is available in the meantime.